packages/cactus-plugin-ledger-connector-ethereum/src/main/json/openapi.json

```diff
@@ -0,0 +1,1301 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Hyperledger Cactus Plugin - Connector Ethereum",
+    "description": "Can perform basic tasks on a Ethereum ledger",
+    "version": "0.0.1",
```

Comment:
There are some new custom-checks in the CI that might start failing because of the version not matching the current release. Just a note for future reference so you know if it happens!

Reply:
@outSH Cheers! I opened this one to combat that issue: https://github.com/hyperledger/cacti/pull/2703